src/components/molecules/DatePicker/stories/DatePicker.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { action } from "@storybook/addon-actions"
+
+import { DatePicker, DatePickerProps } from '../DatePicker'
+
+
+export default {
+  title: 'Molecules/DatePicker',
+  component: DatePicker,
+  argTypes: {
+    register: {
+      control: { type: 'disabled' }
+    },
+  },
+}
+
+export const Basic = (args: DatePickerProps) => (
+  <DatePicker  {...args} onDateChange={(timestamp) => { console.log(timestamp) }} />
```

Comment:
console log statement is probably not the correct method call ;) aren't there storybook actions or somthing like that?

Reply:
Fixed by using an `action` 👍